announcements/2020-Week-38/announcement.md

```diff
@@ -0,0 +1,8 @@
+# :zap: This Week in Provider-Independent Announcements (September 18, 2020):zap:
+
+## Giant Swarm Catalog (Managed Apps)
+
+1. [EFK Stack app v0.3.1](https://github.com/giantswarm/efk-stack-app/blob/master/CHANGELOG.md#031---2020-09-11) adds Giant Swarm monitoring annotations and labels to improve observability.
+
```

Comment:
ping @giantswarm/sig-ux 

```suggestion

## User interface releases

1. [Happa 0.10.39](https://github.com/giantswarm/happa/releases/tag/v0.10.39) contains several fixes and changes related to installing and updating apps. A cluster creation indicator has been added.
2. [gsctl 0.24.2](https://github.com/giantswarm/gsctl/releases/tag/0.24.2) adds an options to silence warnings of insecure usage of non https endpoint urls.
3. [kubectl-gs 0.6.1](https://github.com/giantswarm/kubectl-gs/releases/tag/v0.6.1) disallows templating clusters with invalid versions.

```

Reply:
```diff
- 2. [gsctl 0.24.2](https://github.com/giantswarm/gsctl/releases/tag/0.24.2) adds an options to silence 
+ 2. [gsctl 0.24.2](https://github.com/giantswarm/gsctl/releases/tag/0.24.2) adds an option to silence 
warnings of insecure usage of non https endpoint urls.
```

Otherwise LGTM 👍🏻